web/html/src/components/input/Radio.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+const React = require('react');
+const { InputBase } = require('./InputBase');
+
+type Props = {
+  items: Array<{label: string, value: string}>,
+  horizontal?: boolean,
+  inputClass?: string,
+} & InputBase.Props;
+
+function Radio(props: Props) {
+  const {
+    items,
+    inputClass,
+    ...propsToPass
+  } = props;
+  return (
+    <InputBase {...propsToPass}>
+      {
+        ({
+          setValue,
+          onBlur,
+        }) => {
+          const onChange = (event: Object) => {
+            setValue(event.target.name, event.target.value);
+          };
+          const WrapperElem = props.horizontal ? "span" : "div";
+          return (
+            <>
+              {
+                props.items.map(({label, value}) =>
+                                 <WrapperElem>
+                                   <input
+                                      style={{marginRight: "5px"}}
+                                      type="radio"
+                                      name={props.name}
+                                      value={value}
+                                      checked={props.value === value}
+                                      className={inputClass}
+                                      onBlur={onBlur}
+                                      onChange={onChange} />
+                                   <label style={{marginRight: "5px"}}>
```

Comment:
maybe a "vertical-align: middle" CSS rule would help having the radio button and the text label vertically aligned?

Reply:
Tried it, but it either didn't help or I did something wrong. Feel free to add any commit, if you want to change it.

Reply:
![C3HaXnWWIAE-Ia1](https://user-images.githubusercontent.com/1140720/62861294-e5842780-bcfa-11e9-9013-11feb94cf239.jpeg)

I will see what I can do here :)

Reply:
(LOL)
Thanks!